README.md

```diff
@@ -1,3 +1,74 @@
-# complex
+# complex #
 
 [![windows](https://github.com/BlueQuartzSoftware/complex/actions/workflows/windows.yml/badge.svg)](https://github.com/BlueQuartzSoftware/complex/actions/workflows/windows.yml) [![linux](https://github.com/BlueQuartzSoftware/complex/actions/workflows/linux.yml/badge.svg)](https://github.com/BlueQuartzSoftware/complex/actions/workflows/linux.yml) [![macos](https://github.com/BlueQuartzSoftware/complex/actions/workflows/macos.yml/badge.svg)](https://github.com/BlueQuartzSoftware/complex/actions/workflows/macos.yml) [![clang-format](https://github.com/BlueQuartzSoftware/complex/actions/workflows/format_push.yml/badge.svg)](https://github.com/BlueQuartzSoftware/complex/actions/workflows/format_push.yml)
+
+## Introduction ##
+
+`complex` is a rewrite of the [SIMPL](https://www.github.com/bluequartzsoftware/simpl) library upon which [DREAM3D](https://www.github.com/bluequartzsoftware/dream3d) is written. This library aims to be fully C++17 compliant, removes the need for Qt5 at the library level and brings more flexible data organization and grouping options.
+
+## Prerequisites ##
+
+In order to compile `complex` you will need a C++17 compiler suite installed on your computer.
+
++ Compiler
+  + Windows Visual Studio 2017 v141 toolset
+  + macOS 10.15 and Xcode 12.4 or higher
+  + Linux with GCC Version 9.0 or higher or clang.
+
+## Install vcpkg ##
+
+The `complex` project uses the [vcpkg](https://www.github.com/microsoft/vcpkg) to manage it's dependent libraries. If this is not already installed on your system then you will need to download and compile it.
+
+### Windows ###
+
+Clone the vcpkg repository into a location that it will be used from. inside your home directory or at `C:/vcpkg` is a reasonable spot. *DO NOT USE A PATH WITH SPACES IN ANY OF THE FOLDERS*.
+
+```(lang-console)
+cd C:/Users/[USERNAME]/Applications
+git clone ssh://git@github.com/microsoft/vcpkg
+```
+
+The `bootstrap-vcpkg.bat` file should be run automatically by CMake the first time. This will create the `vcpkg.exe` file. Additionally CMake should automatically find `vcpkg.exe`. If CMake does not find it, you may need to add it to your `PATH` variable.
+
+### MacOS/Linux ###
+
+Select an install location (the example below uses `/Users/Shared/DREAM3DNX_SDK/`) and then clone, build and install vcpkg.
+
+```(lang-console)
+mkdir -p /Users/Shared/DREAM3DNX_SDK/
+cd /Users/Shared/DREAM3DNX_SDK/
+git clone https://github.com/microsoft/vcpkg
+cd vcpkg
+./bootstrap-vcpkg.sh -disableMetrics
+export PATH=/Users/Shared/DREAM3DNX_SDK/vcpkg:$PATH
```

Comment:
This isn't required and should match the Windows section.

Reply:
Done.